src/elements/content-preview/preview-header/index.js.flow

```diff
@@ -1,2 +1,2 @@
 // @flow
-export { default } from './PreviewHeader';
+export { default } from './PreviewHeader';
```

Comment:
Revert this change. Only revert this specific file. Do not do anything else, please.